src/tools/react-native.ts

```diff
@@ -176,3 +176,41 @@ export async function renameReactNativeApp(
     }),
   )
 }
+
+export async function replaceMaestroBundleIds(
+  toolbox: GluegunToolbox,
+  oldBundleIdentifier: string,
+  newBundleIdentifier: string,
+) {
+  const { parameters, filesystem, print } = toolbox
+  const { path } = filesystem
+
+  // debug?
+  const debug = Boolean(parameters.options.debug)
```

Comment:
`Boolean` unfortunately treats any string as truthy, even if the user passes `--debug false`, because the string 'false' is a truthy value.

Ideally this would be handled up stream, but I ended up included a utility to handle this

```suggestion
  const debug = boolFlag(parameters.options.debug)
```

Reply:
Oh interesting ok, I see that used in `new.ts` but I had copied this line from the other rename utility function above, will fix both!